scripts/jaseci.yaml

```diff
@@ -18,8 +49,9 @@ metadata:
 data:
   prod_up: |-
     apt update; apt -y upgrade; apt -y install --no-install-recommends git g++;
-    git clone https://github.com/Jaseci-Labs/jaseci.git
+    git clone https://github.com/Gorgeous-Patrick/jaseci.git;
```

Comment:
We will need to change this to the main jaseci repo when this is ready to be merged.